### DISCUSSION TITLE
Cannot use `kitty` or `sixel` on Windows

### DISCUSSION THREAD

**Discussion by BatmanAoD** [wezterm/wezterm#3744] [Q&A]
I'm trying to see if any terminals available on Windows support the `kitty` or `sixel` protocols, so I installed `chafa` (both as an executable and as a program in WSL) to try printing an image.

In both PowerShell (on the host) and Bash on WSL, neither `chafa -f kitty <img>` nor `chafa -f sixel <img>` produces an image.

Is there any image support that works natively in Windows? (I'm guessing that running WezTerm itself as a [WSL GUI app](https://learn.microsoft.com/en-us/windows/wsl/tutorials/gui-apps) would work, but I prefer to run my GUI apps on the host.)

**Comment by wez** [comment]
Take a look through:
* https://github.com/wez/wezterm/issues/1673

**Reply by BatmanAoD** [reply]
Ah, sorry, I did do an issue search and somehow missed that!
